package.json

```diff
@@ -60,8 +60,9 @@
   },
   "dependencies": {
     "axios": "^0.18.0",
+    "babel-polyfill": "^6.26.0",
     "core-js": "^2.5.5",
-    "fast-xml-parser": "^3.9.10",
+    "fast-xml-parser": "~3.0.0",
```

Comment:
how bad, that we have to roll back the version of the xml-parser.
did you double checked, if the options of the parser doesn't changed between these versions?

Reply:
I found the other way of solving the problem. Please look at newer commit.